runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoOverlay.java

```diff
@@ -154,8 +304,12 @@ else if (opponent instanceof Player)
 				// This is the reverse of the calculation of healthRatio done by the server
 				// which is: healthRatio = 1 + (healthScale - 1) * health / maxHealth (if health > 0, 0 otherwise)
 				// It's able to recover the exact health if maxHealth <= healthScale.
-				int health = 0;
-				if (lastRatio > 0)
+
+				//raidOpponentHealth is non-zero if the raid health overlay is active;
+				//The max and min health don't have to be calculated then
+				int health = raidOpponentHealth;
```

Comment:
What about when the health overlay isn't active?

Reply:
By now I've changed this to `int health = opponentMaxHealth`, so that it only runs when the overlay actually isn't active. Previously it could also run when the opponent simply had zero health. 

But to answer your question: Both the opponenthealth and raidopponenthealth are zero when the overlay isn't active.